src/components/Pokemons.vue

```diff
@@ -29,8 +29,14 @@ export default {
 </script>
 
 <style scoped>
-#wrapper {
-  display: flex;
-  flex-wrap: wrap;
-}
-</style>>
+  #wrapper {
+    display: grid;
+    flex-wrap: wrap;
+    grid-template-columns: repeat(auto-fit, 200px);
+    grid-gap: 20px;
+    justify-content: space-evenly;
+    margin: 80px auto 0;
```

Comment:
Adding a bit of extra margin at the bottom of this wrapper could make it a bit more comfortable to see the latter pokemon (feel free to pick whatever value you want)

```suggestion
    margin: 80px auto 15vh;
```

### Before
![image](https://user-images.githubusercontent.com/21317379/85205347-c854b800-b312-11ea-88c8-8ddfacd8a9ff.png)

### After
![image](https://user-images.githubusercontent.com/21317379/85205350-d0145c80-b312-11ea-8ddc-fa9e3cbfc054.png)